src/core/load-fragment.c

```diff
@@ -305,6 +305,43 @@ int config_parse_unit_path_printf(
         return config_parse_path(unit, filename, line, section, section_line, lvalue, ltype, k, data, userdata);
 }
 
+int config_parse_colon_separated_paths(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        _cleanup_strv_free_ char **l;
+        char **s = data;
+        char **p;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        if (isempty(rvalue))
+                mfree(*s);
+
```

Comment:
I have the suspicion the paths listed here should support specifier expansion. i.e. do a loop around etxract_first_word() and then expand specifiers before adding the strings to the list.